src/opentimelineio/clip.h

```diff
@@ -21,12 +34,21 @@ class Clip : public Item
         std::string const&         name            = std::string(),
         MediaReference*            media_reference = nullptr,
         optional<TimeRange> const& source_range    = nullopt,
-        AnyDictionary const&       metadata        = AnyDictionary());
-
-    void set_media_reference(MediaReference* media_reference);
+        AnyDictionary const&       metadata        = AnyDictionary(),
+        std::string const&         active_media_reference =
+            MediaRepresentation::default_media);
 
+    void            set_media_reference(MediaReference* media_reference);
     MediaReference* media_reference() const noexcept;
 
+    using MediaReferences = std::map<std::string, MediaReference*>;
+
+    MediaReferences media_references() const noexcept;
+    void set_media_references(MediaReferences const& media_references) noexcept;
+
+    std::string active_media_reference() const noexcept;
+    void set_active_media_reference(std::string const& new_active) noexcept;
```

Comment:
This might be a bit nitpick, but `active_media_reference` reads as if the value should be a `MediaReference` instead of a string. If we were being totally explicit, we might call it `active_media_reference_key`.

Do others have thoughts on this?

Reply:
Not a nitpick, naming is important. I was hoping to go over all the naming at the TSC meeting on Thursday to make sure everyone is on board with the chosen names.

Reply:
Do we have a consistent nomenclature throughout OTIO for "keys" or "tags" in general? I do agree that `active_media_reference` sounds like it's going to give me access to a `MediaReference` object in some way.

Reply:
We use `key` in SerializableObject::Reader:
```
 bool read(std::string const& key, bool* dest);
 ```
 
 Since AnyDictionary inherits from std::map, it also uses the term `key`:
 https://en.cppreference.com/w/cpp/container/map
 
 The term `tag` is only really used by rapidjson whose API is not directly exposed to the end user.